elimar/templates/logout.html

```diff
@@ -0,0 +1,7 @@
+{% load django_vite %}
```

Comment:
We don't need the django_vite here. Please load the `base` template tho.
```suggestion
{% extends 'base.html' %}
```